packages/data-transport-layer/src/services/server/service.ts

```diff
@@ -106,29 +106,45 @@ export class L1TransportServer extends BaseService<L1TransportServerOptions> {
   private _initializeApp() {
     // TODO: Maybe pass this in as a parameter instead of creating it here?
     this.state.app = express()
+    if (this.options.ethNetworkName) this._initMonitoring()
```

Comment:
Interesting that this passes the linting. My personal style is

```js
if (this.options.ethNetworkName)
  this._initMonitoring()
```

But the linter seems to yell at that wanting curly braces

Reply:
^will try that out

Reply:
TBH I would prefer curly braces. I think if statements without curly braces are a bit weird and we've mostly been enforcing curly braces elsewhere.